sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,196 @@
+    ownership_cnt = 8
+
+    checkpoint_store = TableCheckpointStore.from_connection_string(
+        storage_connection_str, table_name)
+    ownership_list = []
+    for i in range(ownership_cnt):
+            ownership = {}
+            ownership['fully_qualified_namespace'] = fully_qualified_namespace
+            ownership['eventhub_name'] = eventhub_name
+            ownership['consumer_group'] = consumer_group
+            ownership['owner_id'] = 'owner_id'
+            ownership['partition_id'] = str(i)
+            ownership['etag'] = None
+            ownership['last_modified_time'] = None
+            ownership_list.append(ownership)
+
+    newownershiplist = checkpoint_store.claim_ownership(ownership_list)
+    ownership = [{'fully_qualified_namespace': 'test_namespace', 'eventhub_name': 'eventhub', 'consumer_group': '$default',
+     'owner_id': 'Bill', 'partition_id': '0', 'etag': newownershiplist[0]['etag'], 'last_modified_time': newownershiplist[0]['last_modified_time']}]
+    ownership_list = checkpoint_store.claim_ownership(ownership)
+    assert ownership_list[0]['owner_id'] == 'Bill'
+
+    ownership = [{'fully_qualified_namespace': 'test_namespace', 'eventhub_name': 'eventhub', 'consumer_group': '$default',
+     'owner_id': 'Jack', 'partition_id': '0', 'etag': 'W/"datetime\'2021-08-02T00%3A46%3A51.7645424Z\'"', 'last_modified_time': newownershiplist[0]['last_modified_time']}]
+    with pytest.raises(OwnershipLostError) as e_info:
+            checkpoint_store.claim_ownership(ownership)
+
+    ownership = [{'fully_qualified_namespace': 'test_namespace', 'eventhub_name': 'eventhub', 'consumer_group': '$default',
+     'owner_id': 'Jack', 'partition_id': '10', 'etag': 'W/"datetime\'2021-08-02T00%3A46%3A51.7645424Z\'"', 'last_modified_time': newownershiplist[0]['last_modified_time']}]
+    with pytest.raises(OwnershipLostError) as e_info:
+            checkpoint_store.claim_ownership(ownership)
+
+def _claim_and_list_ownership(storage_connection_str, table_name):
+    fully_qualified_namespace = 'test_namespace'
+    eventhub_name = 'eventhub'
+    consumer_group = '$default'
+    ownership_cnt = 8
+
+    checkpoint_store = TableCheckpointStore.from_connection_string(
+        storage_connection_str, table_name)
+    ownership_list = checkpoint_store.list_ownership(
+            fully_qualified_namespace,
+            eventhub_name,
+            consumer_group)
+    assert len(ownership_list) == 0
+
+    ownership_list = []
+
+    for i in range(ownership_cnt):
+            ownership = {}
+            ownership['fully_qualified_namespace'] = fully_qualified_namespace
+            ownership['eventhub_name'] = eventhub_name
+            ownership['consumer_group'] = consumer_group
+            ownership['owner_id'] = 'owner_id'
+            ownership['partition_id'] = str(i)
+            ownership['etag'] = None
+            ownership['last_modified_time'] = None
+            ownership_list.append(ownership)
```

Comment:
if this snippet is something that you use across multiple tests, rather than redefining inside each, you can move this to a helper method

Reply:
yes I will do those changes